src/main/scala/uk/gov/homeoffice/domain/core/email/EmailRepository.scala

```diff
@@ -67,6 +67,20 @@ trait EmailRepository extends Repository with MongoSupport with Logging {
     for { x <- emailCursor } yield Email(x)
   }
 
+  //Excluded html, text and cc from returned query as take up relatively large amount of space when running reports
+  def findEmailSummaryByDateRange(from: DateTime, to: Option[DateTime]): Iterable[Email] = {
```

Comment:
I was wrong. Want to use an Iterator as it is lazy (and note that it can only be used once).

So the code could be:
```scala
def findEmailSummaryByDateRange(from: DateTime, to: Option[DateTime]): Iterator[Email] = {
    val builder = MongoDBObject.empty

    val fields = MongoDBObject("user" -> 1, "emailId" -> 1, "caseId" -> 1, "caseRef" -> 1, "date" -> 1,
    "recipient" -> 1, "subject" -> 1, "status" -> 1, "emailType" -> 1)

    builder += Email.DATE -> dateRangeQuery(Some(from), to)

    collection.find(builder.result(), fields).map(Email(_))
  }
```
and the related test could be:
```scala
"find email summary by date range" in {
      val emailObj = insertEmail()
      val emails = repository.findEmailSummaryByDateRange(now.minusHours(1), Some(now))

      emails.toList must beLike {
        case List(email) =>
          email.emailId mustEqual emailObj.emailId
          email.html must beNull
          email.text must beNull
      }
    }
```